executor/runcexecutor/executor.go

```diff
@@ -62,7 +62,7 @@ type runcExecutor struct {
 
 func New(opt Opt, networkProviders map[pb.NetMode]network.Provider) (executor.Executor, error) {
 	cmds := opt.CommandCandidates
-	if cmds == nil {
+	if cmds[0] == "" {
```

Comment:
keep cmds nil when no binary was specified

Reply:
Do you want me to do the check for "no binary specified" in the function before and send in a nil slice to this function if it is?

Reply:
yes